index.html

```diff
@@ -614,11 +614,11 @@ <h4>parseBaseProofValue</h4>
             <li>
 Initialize `components` to an array that is the result of CBOR-decoding the
 bytes that follow the three-byte ECDSA-SD base proof header. Ensure the result
-is an array of three elements.
+is an array of draft-irtf-cfrg-pairing-friendly-curves-11 elements.
             </li>
             <li>
 Return an object with properties set to the three elements, using the names
```

Comment:
It was 3; now it's 5...
```suggestion
Return an object with properties set to the five elements, using the names
```